### PR TITLE
Create ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60033](https://cumulocity.atlassian.net/browse/MTM-60033)
Original PR: [6775](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6775)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: application for access mapping are getting from application… (#6740) [GRAFT][release/cd] (#6775)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Access mappings for applications are now retrieved from the application
```

File: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60033
+version: 1020.13.1
+---
+application for access mapping are getting from application… (#6740) [GRAFT][release/cd] (#6775)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, access mappings for applications were not properly retrieved from the application itself, which could lead to incorrect access permissions. With this change, access mappings are now correctly fetched from the respective application. This ensures that the appropriate access rights are applied based on the application's configuration, providing a more reliable and secure access control mechanism for users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60033
+version: 1020.13.1
+---
+The list of applications that can be configured for access by users logging in through SSO has been narrowed down to only include applications available to the tenant.
```
Comment (kgol-c8y) [inline content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:17]: The list of applications that can be chosen in Access mapping of SSO configuration has been limited to include only those available to the respective tenant.

Comment (pink-mongoose) [reply content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:17]: Out of interest do we show the applications that are not available to the tenant but grey them out?  Or do we put an information icon to tell the user that if they cannot see the application they are looking for that the tenant does not have access to it?

File: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The list of applications that can be configured for SSO has been narrowed down to applications available to the tenant.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:3]:
```suggestion
title: List of applications available in SSO configuration now limited to applications available to the tenant
```

File: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md (hunk 3 above)
Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md:17]: As I know it simply does not show up in the drop down as explained in the ticket comments https://cumulocity.atlassian.net/browse/MTM-60033

**Pull request merged**